ivy/functional/frontends/jax/numpy/searching_sorting.py

```diff
@@ -83,3 +83,8 @@ def nanargmin(a, /, *, axis=None, out=None, keepdims=None):
     a = ivy.where(nan_mask, ivy.inf, a)
     res = ivy.argmin(a, axis=axis, keepdims=keepdims)
     return ivy.where(ivy.all(nan_mask, axis=axis, keepdims=keepdims), -1, res)
+
+
+@to_ivy_arrays_and_back
+def sort_complex(a):
+    return ivy.sort_complex(a)
```

Comment:
Hi! The problem is that `ivy.sort_complex` does not exist yet. You can implement it as a composition. I recommend reading [this guide](https://lets-unify.ai/ivy/deep_dive/ivy_frontends.html#compositions) and [this one](https://lets-unify.ai/ivy/contributing/open_tasks.html#frontend-apis) to get a better understanding of how to handle this situation. If you have any questions, please let me know! 😄

Reply:
Hi! I have updated this function as a composition of pre-implemented ivy functions. Please can you check again?

Reply:
@ZoeCD

Reply:
Hi! There seem to be some problems with the implementation. The `intelligent-test` are showing the following exception `hypothesis.errors.InvalidArgument: Cannot sample from a length-zero sequence.` And `test-frontend-jax` is showing `TypeError: can't convert complex to float`. Could you check that, please? 🙌